release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -284,6 +284,182 @@ To support this feature, OLM now enforces CRD upgrades are safer by ensuring
 existing CRD storage versions are not missing in the upgraded CRD, avoiding
 potential data loss.
 
+[id="ocp-4-5-operator-api"]
+==== Read-only Operator API (Technology Preview)
+
+The new Operator API is now available as a Technology Preview feature in
+read-only mode. Previously, installing Operators using Operator Lifecycle
+Manager (OLM) required cluster administrators to be aware of multiple APIs,
+including CatalogSources, Subscriptions, ClusterServiceVersions, and
+InstallPlans. This single Operator API resource is a first step towards a more
+simplified experience discovering and managing the lifecycle of Operators in a
+{product-title} cluster.
+
+Currently only available using the CLI and requiring a few manual steps to
+enable, this feature previews interacting with Operators as a first-class API
+object. Cluster administrators can discover previously installed Operators using
+this API in read-only mode.
+
+In a future release of {product-title}, cluster administrators will be able to
```

Comment:
Do we typically promise future features in the release notes? Devils Advocate: what if this changes?

Reply:
Removed!